test/lib/helpers.ts

```diff
@@ -41,12 +42,22 @@ export class Helpers {
 
   async removeRouteResponse() {
     const deleteButtonSelector =
-      '#route-responses-menu .btn-link:not(.doc-link)';
+      '#route-responses-menu .btn-link:not(.doc-link):not(#duplication-button)';
 
     await this.testsInstance.app.client.element(deleteButtonSelector).click();
     await this.testsInstance.app.client.element(deleteButtonSelector).click();
   }
 
+  async duplicateRouteResponse() {
+    const duplicationButtonSelector = '#route-responses-menu #duplication-button';
+    await this.testsInstance.app.client.element(duplicationButtonSelector).click();
+  }
+
+  async removeDuplicatedRouteResponse() {
```

Comment:
I think this helper is a little bit too specific, please select the route response during the test using `selectRouteResponse()` and delete it using the existing `removeRouteResponse()` helper.